lib/rules/asi-safe.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @fileoverview Rule to flag lines dangerous when relying on ASI
+ * @author Glen Mailer
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+module.exports = function(context) {
+
+    function report(node, token) {
+        context.report(node, {
+            line: token.loc.start.line,
+            column: token.loc.start.column
+        }, "Line starting with " + token.value + " continues previous");
```

Comment:
This message needs to be better. error messages should say what is expected or unexpected, this does neither.